software/libre-workspace.yml

```
@@ -0,0 +1,14 @@
+name: "Libre Workspace"
+website_url: "https://www.libre-workspace.org/"
+source_code_url: "https://github.com/Jean28518/libre-workspace"
+description: "modern Cloud-Server for individuals/small organisations, based on Debian Linux, with built in Nextcloud, Matrix, Jitsi, Samba, BorgBackup and a Linux Desktop as Workspace, Extensible."
```

```suggestion
description: "Modern Cloud-Server for individuals/small organisations with built in Nextcloud, Matrix, Jitsi, Samba, BorgBackup and a Linux Desktop as Workspace, Extensible."
```
A shortened description, if there is a better one, or this change doesnt fit, feel free to abandon the correction.